newIDE/app/src/EventsFunctionsExtensionsLoader/MetadataDeclarationHelpers.js

```diff
@@ -1171,6 +1171,42 @@ export const declareObjectPropertiesInstructionAndExpressions = (
   });
 };
 
+/**
+ * Declare the instructions (actions/conditions) and expressions for the
+ * properties of the given events based object.
+ * This is akin to what would happen by manually declaring a JS extension
+ * (see `JsExtension.js` files of extensions).
+ */
+export const declareObjectInternalInstructions = (
+  i18n: I18nType,
+  extension: gdPlatformExtension,
+  objectMetadata: gdObjectMetadata,
+  eventsBasedObject: gdEventsBasedObject
+): void => {
+  // TODO EBO Use full type to identify object to avoid collision.
+  // Objects are identified by their name alone.
+  const objectType = eventsBasedObject.getName();
+
+  objectMetadata
+    .addScopedAction(
+      'SetRotationCenter',
+      i18n._('Center of rotation'),
+      i18n._(
+        'Change the center of rotation of an object relatively to the object origin.'
+      ),
+      i18n._('Change the center of rotation of _PARAM0_: _PARAM1_; _PARAM2_'),
```

Comment:
Set the center of rotation of X to Y;Z (relative to the object origin)

Reply:
I don't mind either but "Change" is used in most action, the one related to positions. Should it be consistent?

Reply:
In this case, it should be `Change the center of rotation of X (relative to the object origin): set to Y;Z`